server/bundles/io.cloudbeaver.server/src/io/cloudbeaver/service/navigator/WebServiceBindingNavigator.java

```diff
@@ -63,6 +64,11 @@ public void bindWiring(DBWBindingContext model) throws DBWebException {
 
             ));
         model.getMutationType()
+            .dataFetcher("navRefreshNode", env -> getService(env).refreshNavigatorNode(
```

Comment:
Two `navRefreshNode` functions in a single service?

Reply:
renamed to `navReloadNode`